### PR TITLE
Enabling detach for offroad scenario.

### PR DESCRIPTION
**Pull request by rajkumar42**
Note: For this to work, "detach all" from visual studio.

### CODE REVIEW

**Comment by chuckries** [comment]
:+1:

File: src/MICore/CommandFactories/clrdbg.cs [hunk 1]
```diff
@@ -43,8 +43,7 @@ public override bool AllowCommandsWhileRunning()
 
         public override bool CanDetach()
```
Comment (gregg-miskelly) [inline src/MICore/CommandFactories/clrdbg.cs]: You can just delete this method now. The default version returns true.

**Comment by gregg-miskelly** [comment]
Otherwise looks good.

**Comment by wesrupert** [comment]
:+1:

**Pull request merged**